src/main/java/org/folio/dew/batch/acquisitions/edifact/CompositePOLineConverter.java

```diff
@@ -452,4 +473,25 @@ private String getLocationCode(Location location) {
     }
     return "";
   }
+
+  private String calculateCostUnitsTotal(Cost cost) {
```

Comment:
if we could not reuse the code, please add coment what method of mod-orders we use here